### PR TITLE
feat(HQM-142): Transactions migration and transaction statuses

### PR DESCRIPTION
**Pull request by devkudasov**
Fixes #

## Proposed Changes

- Migrate transactions from realm to mobx
- Show transaction status
-

## Checklist

- [x] I have read the rules of [**contribution**](https://github.com/haqq-network/haqq-wallet/blob/main/docs/contribution.md#important-steps).

### CODE REVIEW

File: src/models/transaction.ts [hunk 1]
```diff
@@ -28,134 +60,150 @@ export class Transaction extends Realm.Object {
...
-      Logger.captureException(e, 'Transaction.setConfirmed', {
-        receipt: JSON.stringify(receipt),
-      });
+  setConfirmed(id: string, receipt: TransactionReceipt) {
+    const transaction = this.getById(id);
+
+    if (transaction) {
+      transaction.confirmed = true;
+      transaction.fee = calcFee(
+        receipt.effectiveGasPrice ?? 7,
```
Comment (ragozin-nikita) [inline src/models/transaction.ts:201]: `7` should be a constant

File: package.json [hunk 2]
```diff
@@ -214,4 +214,4 @@
     "vm": "vm-browserify"
   },
   "packageManager": "yarn@3.6.0"
-}
\ No newline at end of file
+}
```
Comment (ragozin-nikita) [inline package.json:217]: Why do you have this change?

File: src/models/transaction.ts (hunk 1 above)
Comment (devkudasov) [reply src/models/transaction.ts]: Moved to const DEFAULT_GAS

**Review by vivalaakam — APPROVED** [review]

**Comment by Jaideepthory** [comment]
I hope so it resolved

**Review by ragozin-nikita — APPROVED** [review]

**Pull request merged**